src/VisualStudio/Core/Def/Implementation/Remote/VisualStudioRemoteHostClientProvider.cs

```diff
@@ -51,23 +51,8 @@ private VisualStudioRemoteHostClientProvider(HostWorkspaceServices services)
             _lazyClient = new AsyncLazy<RemoteHostClient>(CreateHostClientAsync, cacheResult: true);
         }
 
-        private async Task<RemoteHostClient> CreateHostClientAsync(CancellationToken cancellationToken)
```

Comment:
not part of this PR.  i'm branched off of Tomas's change so i can use VS without OOP crashing.  Once https://github.com/dotnet/roslyn/pull/46929 goes in, this will be gone.